src/components/health-questionnaire/steps/SecondStepItem.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { Typography } from '@mui/material';
+import { selectActivity } from 'src/redux/slices/healthQuestionnaireSlice';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from 'src/redux/store';
+import { useLocales } from 'src/locales';
+import { ToggleButtonGroupStyled, ToggleButtonStyled } from '../styles';
+
+const SecondStepItem = ({
+  questions,
+  options,
+  onCompletion,
+}: {
+  questions: string[];
+  options: string[];
+  onCompletion: (status: boolean) => void;
+}): JSX.Element => {
```

Comment:
Could you please create type for props before component? as we did for other components